datamonster_api/lib/datamonster.py

```diff
@@ -276,7 +276,9 @@ def _avro_to_df(self, avro_buffer, data_types):
 
         def parse_row(row):
             return {
-                col["name"]: pandas.to_datetime(row[col["name"]])
+                col["name"]: pandas.to_datetime(
+                    row[col["name"]] if row[col["name"]] != "None" else None
```

Comment:
Checking for a hardcoded string seems sketchy here, maybe it makes more sense to try to parse it and then do None if the parsing fails?  Also, I'm not sure we should support misconfigured data.  `None` is clearly not a date.

Reply:
updated